src/main/java/edu/miu/cs/cs544/EAProject/domain/CountryRegion.java

```diff
@@ -11,7 +12,7 @@
 @Entity
 @Table(name = "CountryRegion")
 @NoArgsConstructor
-@Getter
```

Comment:
I just wanna allow some properties to change the value in runtime. That's why I didn't use @Data